packages/styles/src/components/datepicker.scss

```diff
@@ -7,15 +7,15 @@
 
 ngb-datepicker {
   // Conversion for compatibility
-  --bs-light: var(--light);
+  --bs-light: var(--post-light);
```

Comment:
Do we need to invert `--post-light` and `--post-dark` in `%color-background-dark-variables`?